client/my-sites/checkout/composite-checkout/components/wp-checkout-order-summary.js

```diff
@@ -136,7 +148,10 @@ function CheckoutSummaryFeaturesList( props ) {
 	const isJetpackNotAtomic = useSelector(
 		( state ) => isJetpackSite( state, siteId ) && ! isAtomicSite( state, siteId )
 	);
+	const site = useSelector( ( state ) => getSite( state, siteId ) );
 	const { hasMonthlyPlan = false } = props;
+	const isPaidSite = site ? ! isFreePlanProduct( site?.plan ) : false;
```

Comment:
Good work checking if the `site` exists before passing it to `isFreePlanProduct`, since the latter will probably throw if passed `undefined`, but if we know that it exists, then we probably don't also need the optional chaining inside the call (which by itself seems to imply that we _might_ call that function with an undefined argument).

```suggestion
	const isPaidSite = site ? ! isFreePlanProduct( site.plan ) : false;
```